spdm_dump/spdm_dump.h

```diff
@@ -141,11 +141,12 @@ extern char *m_param_out_rsq_cert_chain_file_name;
 
 extern void *m_requester_cert_chain_buffer;
 extern size_t m_requester_cert_chain_buffer_size;
-extern void *m_responder_cert_chain_buffer;
-extern size_t m_responder_cert_chain_buffer_size;
+extern void *m_responder_cert_chain_buffer[SPDM_MAX_SLOT_COUNT];
+extern size_t m_responder_cert_chain_buffer_size[SPDM_MAX_SLOT_COUNT];
 extern void *m_dhe_secret_buffer;
 extern size_t m_dhe_secret_buffer_size;
 extern void *m_psk_buffer;
 extern size_t m_psk_buffer_size;
 
+extern uint8_t m_use_slot_id;
```

Comment:
is this responder slot ? or requester slot?

Reply:
This is responder slot id. I have changed it to `m_res_slot_id` and added `m_req_slot_id` too.